pom.xml

```diff
@@ -99,6 +167,7 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.core</artifactId>
             <version>6.0.0</version>
+            <scope>compile</scope>
```

Comment:
same here...should be be provided?

Reply:
@arvindn05, this was already compile scope in the earlier code and shall change to provided.